libs/blocks/gnav/gnav.css

```diff
@@ -1183,6 +1183,10 @@ header .app-launcher {
     box-sizing: border-box;
   }
 
+  .gnav .gnav-logo svg {
+    width: 25px;
```

Comment:
Since its parent already has a set width, I think `100%` should be enough. Did a quick test on Saucelabs on Safari v15 to check it out:
<img width="1311" alt="Screenshot 2023-11-10 at 11 25 27" src="https://github.com/adobecom/milo/assets/11267498/d07e92a5-8db8-44a8-8ea8-8377d717430f">

Reply:
Updated